Client/Frontend/Strings.swift

```diff
@@ -1053,6 +1053,29 @@ extension String {
                 comment: "In the settings menu, in the Toolbar customization section, this label indicates that selecting this will make the toolbar appear at the bottom of the screen.")
         }
 
+        public struct Autoplay {
+            public static let Autoplay = MZLocalizedString(
+                "Settings.Autoplay.SettingsTitle",
```

Comment:
For each key we should add the version this was introduced in, I think this work could be part of v111 (to be confirmed but if you want you can make the change already). So in the case of the first key, it would be `"Settings.Autoplay.SettingsTitle.v111"`

Reply:
thanks, i see that pattern in others now. done.